cd/manager/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ekidd/rust-musl-builder:stable as builder
+
+RUN USER=root cargo new --bin ceramic-cd-manager
+WORKDIR ./ceramic-cd-manager
+COPY ./Cargo.lock ./Cargo.lock
+COPY ./Cargo.toml ./Cargo.toml
+RUN cargo build --release
+RUN rm src/*.rs
+
+ADD . ./
+
+RUN rm ./target/x86_64-unknown-linux-musl/release/deps/ceramic_cd_manager*
```

Comment:
You lost me here.. we delete the binary then build it again in the best step? Why?

Reply:
Please see [this](https://github.com/3box/pipeline-tools/pull/4#discussion_r914288024) comment. We're setting up the most cacheable steps first, then the least cacheable ones last (i.e. those that end up changing the source code).